package.json

```diff
@@ -45,62 +45,18 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.6.2",
```

Comment:
As Aaron mentioned, the goal for having these dev dependencies up at the root was to help enforce consistent versions of these common tools. Given that dev deps don't get installed by cli users, is there a harm to leaving these here? Or is there an alternative way to help enforce versions?

Reply:
Well, it takes responsibility of the package maintainers to install and update the packages they depend on. Different scenarios:

- Even without installing (declaring in its own package.json) package x can be used by package y. On releasing of package y package x is not contained in package.json of y. Nor any package-lock or yarn-lock is included in the packages bundle. What happens?
- Packages that don't have anything to do with babel (like the typescript ones) share this dependency. For what reason?
- Impossible to resolve peer dependencies when having inherited depencies like: ```
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-syntax-flow@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-arrow-functions@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-block-scoping@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-classes@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-computed-properties@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-destructuring@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-exponentiation-operator@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-flow-strip-types@7.0.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-for-of@7.8.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-function-name@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-literals@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-modules-commonjs@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-object-assign@7.0.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-parameters@7.8.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "workspace-aggregator-d2b6cd31-034c-43c4-a4b0-26299387134e > @celo/mobile > metro-react-native-babel-preset > @babel/plugin-transform-react-display-name@7.0.0" has unmet peer dependency "@babel/core@^7.0.0-0".
```